src/cli/cmd.js

```diff
@@ -0,0 +1,116 @@
+'use strict';
+
+var path = require('path');
+var winston = require('winston');
+var async = require('async');
+var db = require('../database');
+var plugins = require('../plugins/data');
+
+function parseCommands(pluginList, callback) {
+	var commands = [];
+
+	pluginList.forEach((pluginData) => {
+		var pluginCommands = pluginData.commands || [];
+
+		pluginCommands.forEach((cmdData) => {
+			var pluginName = pluginData.id;
+			var cmdName = pluginName.replace('nodebb-plugin-', '') + ':' + cmdData.cmd;
+
+			commands.push({
+				plugin: pluginName,
+				name: cmdName,
+				description: cmdData.description,
+				options: cmdData.options,
+				action: (cmdArgs, done) => {
+					var libraryFile = cmdData.library ? cmdData.library : pluginData.library;
+					var scriptFile = path.resolve(pluginData.path, libraryFile);
+					var call = require(scriptFile)[cmdData.method];
+					call(cmdArgs, done);
+				},
+			});
+		});
+	});
+	callback(null, commands);
+}
+
+function printCommandList(commands) {
+	console.log('\n');
+	if (commands.length) {
+		commands.forEach((cmd) => {
+			console.log('\x1b[32m' + cmd.name + '\x1b[0m \t\t\t ' + cmd.description);
```

Comment:
You should use the `colors` module in instances where you want styles console text like so:

    'string'.green

Reply:
ddb49a7